.bazelrc

```diff
@@ -8,11 +10,15 @@
 startup --host_jvm_args=-Xmx512m
 
 build --workspace_status_command=tools/bazel_get_workspace_status
-# enable path normalization by default. See https://github.com/envoyproxy/envoy/pull/6519
-build --define path_normalization_by_default=true
 build --host_force_python=PY2
 
+# Link (statically) against libstdc++.
+build --action_env=BAZEL_LINKLIBS=-l%:libstdc++.a
+build --action_env=BAZEL_LINKOPTS=-lm:-static-libgcc
```

Comment:
I synchronized the other options with upstream while at it, but those lines are the real fix.